app/src/processing/app/contrib/StatusPanel.java

```diff
@@ -174,11 +173,11 @@ public void actionPerformed(ActionEvent e) {
                               .addComponent(updateLabel))
                   .addComponent(updateButton).addComponent(removeButton)));
 
-    layout.linkSize(SwingConstants.HORIZONTAL, installButton, progressBarPanel,
```

Comment:
Please don't reformat like this. See the style guidelines (and where that's missing things, see the rest of the code). This change makes things less readable.